pkg/network/dns/snooper.go

```diff
@@ -127,28 +126,6 @@ func (s *socketFilterSnooper) GetDNSStats() StatsByKeyByNameByType {
 	return s.statKeeper.GetAndResetAllStats()
 }
 
-// GetStats returns stats for use with telemetry
-func (s *socketFilterSnooper) GetStats() map[string]int64 {
-	stats := s.cache.Stats()
-
-	for key, value := range s.source.Stats() {
-		stats[key] = value
-	}
```

Comment:
this pulled relevant telemetry from the windows driver and added it to the reported DNS stats. From what I can tell, the new dns snooper telemetry no longer reports this information. Could it be added back?

Reply:
I added this back and per our DM, added a variety of windows specific telemetry to the changes